### PR TITLE
Feature/conjugator options

### PR DESCRIPTION
**Pull request by dunetna**
# Description
Add options to conjugator.
# How to test
Go to ```/words/<verb>/conjugation```, test different options and check that conjugations table changes according to selected options.

### CODE REVIEW

**Review by slamora — CHANGES_REQUESTED** [review]
Everything is working as expected! Just a two comments:
1. Regarding to input style, AFAIK, will be more flexible follow the approach of hidding default item and customize a new one [as suggested in W3C example](https://www.w3schools.com/howto/howto_css_custom_checkbox.asp).
2. Remove "Grafía" option, as DGPL should we always used to be homogeneous with the rest of the content.

File: linguatec_lexicon_frontend/forms.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    CHERUNDIO_CHOICES = (
+        ('completa', 'completa (rematada en -o)'),
+        ('apocopada', 'apocopada (no rematada en -o)')
+    )
+
+    grafia = forms.ChoiceField(
+        label="Grafía", choices=GRAFIA_CHOICES, widget=forms.RadioSelect
+    )
+    participio = forms.ChoiceField(
+        label="Rematanza d'os participios", choices=REMATANZA_CHOICES, widget=forms.RadioSelect
```
Comment (slamora) [inline linguatec_lexicon_frontend/forms.py:33]:
```suggestion
        label="Rematanza d'os partizipios", choices=REMATANZA_CHOICES, widget=forms.RadioSelect
```

**Review by slamora — CHANGES_REQUESTED** [review]

**Comment by dunetna** [comment]
> Everything is working as expected! Just a two comments:
> 
>     1. Regarding to input style, AFAIK, will be more flexible follow the approach of hidding default item and customize a new one [as suggested in W3C example](https://www.w3schools.com/howto/howto_css_custom_checkbox.asp).

Done at https://github.com/lenguasdearagon/aragonario-frontend/pull/8/commits/b24379c21e85753e513da4c46100ec9384fe2499

>     2. Remove "Grafía" option, as DGPL should we always used to be homogeneous with the rest of the content.
Done at https://github.com/lenguasdearagon/aragonario-frontend/pull/8/commits/39023c49ee0296eb225bd0dcbfea5bdb551c658d

**Review by slamora — CHANGES_REQUESTED** [review]
Why don't use col-12 and col-3 to use all the width available?

**Comment by dunetna** [comment]
> Why don't use col-12 and col-3 to use all the width available?

Because I'm not the best front-end developer XDD
Fixed using col-12 and CSS right margin :-)

**Review by slamora — APPROVED** [review]

**Pull request merged**